modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientExperimentalCodegen.java

```diff
@@ -392,8 +401,13 @@ private void fixModelImports(Set<String> imports) {
                     }
                 }
 
-                // fix the imports that each model has, change them to absolute
-                fixModelImports(cm.imports);
+                /*
+                 * Moved this functionality into toModelImport.
+                 * 
+                 * Now the imports are one level higher, so the template has also been altered.
+                 */
+                //// fix the imports that each model has, change them to absolute
+                //fixModelImports(cm.imports);
```

Comment:
Can you delete this comment. We don't need it going forward.

Reply:
Obviously deleting it is not a problem. But I left it in on purpose. I tried my best not to interfere with the experimental client, as I consider its not my place to interfere. I left this comment in to make it clear to the participants of that sub project, the nature of a speculative change that could effect them. This type definition here is an experimental proposal on top of an experiments design. It was not clear to me what the long term strategy was, nor if this change should be included here, or not.
You could could still choose for the new generator to go in a different direction. For me the fact that this PR effects the experimental generator make the decision to keep it, or not ambiguous.
So, I will leave this for you to confirm, or not.

Reply:
The experimental generator can have breaking changes in it.
If we added comments like this when we made breaking changes we would have too many comments in the code base. I appreciate the thought, but feel that it adds unnecessary documentation to the code base. Please delete it.

Reply:
OK, on the list